tools/topology/topology2/production/tplg-targets-hda-generic.cmake

```diff
@@ -16,6 +16,8 @@ EFX_HDA_MIC_TDFB_PARAMS=line2_pass,EFX_HDA_MIC_DRC_PARAMS=passthrough"
 # HDA topology with mixer-based pipelines for HDA and
 # passthrough pipelines for HDMI and
 # 2 or 4 DMIC, no NHLT blob included in topology
+"sof-hda-generic\;sof-hda-generic-1ch\;HDA_CONFIG=mix,NUM_DMICS=1,PDM0_MIC_B_ENABLE=false"
```

Comment:
Should PDM0_MIC_B_ENABLE be 0 instead of false?
And do we need DMIC0_ENHANCED_CAPTURE for the topologies?

Reply:
Yes, 0 / 1 seems to be more common. Then maybe better to change.

I dropped DMIC0_ENHANCED_CAPTURE because the beamformer doesn't support mono. The 1ch version would need a topology variant with only DRC. I'd propose to fix this in two steps, first without DRC, and after that add the DRC.